README.md

```diff
@@ -140,17 +140,17 @@ packages for Visual Studio 2012.
 
 Triton is strongly powered by [Quarkslab](https://quarkslab.com) for years but also by several strong contributors:
 
-* **Alberto Garcia Illera** - Salesforce
-* **Alexey Vishnyakov** - ISP RAS
+* **Alberto Garcia Illera** - Cruise Automation
```

Comment:
I think alberto was at Salesforce when committing for Ponce. (cc @illera88)

Reply:
I was